mk/rt.mk

```diff
@@ -384,7 +384,11 @@ COMPRT_OBJS_$(1) += emutls.o
 endif
 
 ifeq ($$(findstring msvc,$(1)),)
+
+ifeq ($$(findstring freebsd,$(1)),)
```

Comment:
I believe these conditions are Makefile-speak vor "not contains" (`findstring` returns the found string, which is then compared to the empty string, as far as I know).